runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -297,6 +306,51 @@ private void playerSkillLookup(ChatMessageType type, SetMessage setMessage, Stri
 		}
 	}
 
+	/**
+	 * Calculates a simple equation and changes the original message to the
+	 * response.
+	 *
+	 * @param messageNode The chat message containing the command.
+	 * @param eqn         The equation to calculate.
+	 */
+	private void calculate(MessageNode messageNode, String eqn)
+	{
+		final ChatMessageBuilder builder = new ChatMessageBuilder();
+		
+		try
+		{
+			final Expression expr = new ExpressionBuilder(eqn).build();
+			final String result = String.valueOf(expr.evaluate());
+			builder.append(ChatColorType.NORMAL)
+				.append("Calc: ")
+				.append(eqn)
+				.append(" = ")
+				.append(ChatColorType.HIGHLIGHT)
+				.append(result);
+		}
+		catch (IllegalArgumentException ex)
```

Comment:
cant you use multicatch and `.append(" Error: ").append(ex.getMessage())` ?

Reply:
I think it's better to have a single catch for each (IllegalArgumentException is a subclass of RuntimeException). Updated builder code accordingly to indicate which case it is.

Reply:
Well, you are catching both, and you can get data about the message to display from exception, I dont think 2 catch blocks are necessary at all.

Reply:
You're right. Done.